Vagrantfile

```diff
@@ -59,6 +59,10 @@ EOM
         apt-get install --yes nodejs
         npm install --global npm@latest
         npm install --global bower
-
+        npm install -g grunt-cli
+        npm install --global grunt
+        cd mbofui
+        npm install
```

Comment:
We talked about this a little this afternoon.  Looking more closely, I think this line, `npm install`, is unnecessary.  It's already run on line 60, above.